coralproject/talk
# Heroku Build Fix

## What does this PR do?

Pins version of NodeJS to ^12 from >=12. This resolves an issue in JSDOM when deploying to Heroku.

## How do I test this PR?

Install on Heroku (or with NodeJS version ^13), and note that the server fails to start by an error with NodeJS and JSDOM.
